atox/src/main/kotlin/ActionReceiver.kt

```diff
@@ -30,8 +30,16 @@ import ltd.evilcorp.domain.tox.PublicKey
 import ltd.evilcorp.domain.tox.Tox
 
 const val KEY_TEXT_REPLY = "text_reply"
-const val KEY_CALL = "accept_or_reject_call"
 const val KEY_CONTACT_PK = "contact_pk"
+const val KEY_ACTION = "key_action"
```

Comment:
Didn't want to touch unrelated lines since the strings only map from the NotificationHelper to the ActionReceiver and the actual values are unimportant (but `key_<thing>` just feels better, y'know?), but will change it to use the current unprefixed convention. :P

Reply:
The `key_<thing>` is indeed feels better. But it's important to keep the convention. So maybe add a commit before that which renames all of the other names to `key_<thing>`?